types/ember/index.d.ts

```diff
@@ -1950,6 +1950,7 @@ declare module 'ember' {
              * Transition the application into another route. The route may
              * be either a single route or route path
              */
+            transitionTo(name: string): Transition;
```

Comment:
Are you sure this is needed? You should be able to call the vararg overload with 1 argument.

Reply:
Yep, you're right, removing.